src/context.ts

```diff
@@ -1,13 +1,15 @@
 import { TypeMap } from "./types";
 import { Utils } from "./main";
 import { Options } from "./options";
-import { FileDescriptorProto } from "ts-proto-descriptors";
+import { FileDescriptorProto as TsProtoFileDescriptorProto } from "ts-proto-descriptors";
+import { FileDescriptorProto as GoogleFileDescriptorProto } from "google-protobuf/google/protobuf/descriptor_pb";
```

Comment:
What's different about the `GoogleFileDescriptorProto`? Can we get the `ts-proto-descriptors` to create the same types that it does?